README.md

```diff
@@ -508,17 +507,19 @@ Read more at [Proguard](https://www.guardsquare.com/en/proguard/manual/examples)
 
 #### SharedPreferences
```

Comment:
Wouldn't be better to wrap that up in 1 topic? For instace, how to retain information. 1) Through intent, 2) Through SharedPreferences explaining that it stores key/value and just just and 3) ContentProviders, saying that if we want to have more complex data stored that would be an option but there is the drawback of writing too much boiler plate code.